App/Services/Notifications.ts

```diff
@@ -0,0 +1,129 @@
+import PushNotification from 'react-native-push-notification'
+import * as TextileTypes from '../Models/TextileTypes'
+
+export interface NotificationsPayload {
+  title: string,
+  message: string,
+  category: string
+}
+
+function getAddressable (notification: TextileTypes.Notification) : string {
+  let actor = notification.actor_username || notification.actor_username.substring(0, 6)
+  if (!actor || actor === '') {
+    actor = 'Someone'
+  }
+  return actor
+}
+
+function notificationToPayload(notification: TextileTypes.Notification): NotificationsPayload {
+  const category = TextileTypes.NotificationType[notification.type] as string
+  switch (notification.type) {
+    case(0): {
```

Comment:
Yea, think these should by typed enums if you update `TextileTypes` to use the enum instead of `number`.